admin_guide/taints_tolerations.adoc

```diff
@@ -0,0 +1,329 @@
+[[admin-guide-taints]]
+= Taints and tolerations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _taint_ allows a node to refuse to allow a pod to be scheduled unless that pod has a _toleration_ that allows the scheduling.
+
+In short, *taints* go on nodes through the node specification (NodeSpec) and *tolerations* go on pods through the pod specification (PodSpec). A taint on a node
+means _repel all pods that do not tolerate the taint_.
+
+
+A taint consists of a key, value, and effect. As a command-line argument, it is expressed as key=value:effect.
+
+.Taint components
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`key`
+|The `key` is any string, up to 253 characters. The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+
+|`operator`
+|
+
+|`value` 
+| The `value` is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+
+|`effect`
+
+|[frame=none]
+[cols="2,3"]
+!====
+!`NoSchedule`!The scheduler will not schedule a pod without this toleration.
+!`PreferNoSchedule` !The scheduler might schedule a pod without this toleration. But, will try not to.
+!`NoExecute` !A pod without this toleration will be evicted from the node, if present when the taint is added ot the node.
+!====
+|===
+
+
+Note the following:
+
+* The `Operator` parameter defaults to `Equal` if not specified.
+* An empty `key` parameter with operator `Exists` matches all keys and all values.
+* An empty `effect` parameter matches all effects.
+
+A toleration matches a taint: 
+
+* If the `operator` parameter is set to `Exists`:
+** the `key` parameters are the same
+** the `effect` parameters are the same
+
+* If the `operator` parameter is set to `Equal`:
+** the `key` parameters are the same
+** the `effect` parameters are the same
+** the `value` parameters are the same
+
+
+== Adding a Taint to an Existing Node
```

Comment:
A toleration is added using the pod spec?? Do you have an example pod spec w/ toleration I could use?